applications/StructuralMechanicsApplication/custom_constitutive/generic_small_strain_isotropic_plasticity.cpp

```diff
@@ -532,7 +560,7 @@ Matrix& GenericSmallStrainIsotropicPlasticity<TConstLawIntegratorType>::Calculat
         Matrix constitutive_matrix;
         this->CalculateElasticMatrix(constitutive_matrix, rParameterValues);
 
-        array_1d<double,VoigtSize> tmp = prod(constitutive_matrix, strain_vector - mPlasticStrain);
+        const array_1d<double,VoigtSize> tmp = prod(constitutive_matrix, strain_vector - mPlasticStrain);

```

Comment:
this is computing the "old" plastic strain, with the savd value of mPlasticStrain, as changed in the FinalizeSolutionStep. Is that what you want?

Reply:
@AlejandroCornejo

Reply:
I think that @loumalouomega wants to compute the equivalent plastic strain (double type) that depends on the stress_vector, plastic_strain_vector and the uniaxial_stress (double) so I think that, if the implementation is correct, should be fine... Do you agreee @loumalouomega ?

Reply:
Yes